### PR TITLE
Rebalance Design Work

### PR DESCRIPTION
**Pull request by UsamaSaleem14**
1. Design of rebalance page

### CODE REVIEW

File: components/Rebalance/RebalanceController.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import * as React from 'react'
+import RebalanceView from './RebalanceView'
```
Comment (TBomberman) [inline components/Rebalance/RebalanceController.tsx:2]: We need to be using custom Hooks.

Comment (UsamaSaleem14) [reply components/Rebalance/RebalanceController.tsx]: Done

**Comment by TBomberman** [comment]
Ok with me @Pochetes

File: hooks/useRebalanceHandler.tsx [hunk 2]
```diff
@@ -0,0 +1 @@
+export const useRebalanceHandler = () => {}
```
Comment (Pochetes) [inline hooks/useRebalanceHandler.tsx:1]: Did you mean to do this? Or are you adding the logic for next PR? If so, remove this file for now

**Review by Pochetes — COMMENTED** [review]
For what is this page used for? There doesn't seem to be a user-led route to this page other than the `localhost:3000/positions/rebalance` url, let me know before merging

File: hooks/useRebalanceHandler.tsx (hunk 2 above)
Comment (TBomberman) [reply hooks/useRebalanceHandler.tsx:1]: Since this PR is only for th UI, this file is just a placeholder for the future backend logic.

**Comment by TBomberman** [comment]
> For what is this page used for? There doesn't seem to be a user-led route to this page other than the `localhost:3000/positions/rebalance` url, let me know before merging

Daniel mentions there should be a way to rebalance a user's collateral from one token to the other.

**Review by Pochetes — APPROVED** [review]

**Pull request merged**